docs/v1/use-cases.md

```diff
@@ -1,60 +1,69 @@
 # Service Catalog Use Cases
 
-TODO: add glossary
-TODO: scrub doc post glossary
-TODO: add CF SB API as the lingua franca for blackbox services
+* TODO: scrub doc post glossary
+* TODO: add CF SB API as the lingua franca for blackbox services
+
+## Personas
+
+* User
```

Comment:
I think @jwforres is right - let's scrub user and dissolve the 'user' use cases in one of other the existing personas or a new one.